### PR TITLE
ENTESB-17613 update bouncycastle artifacts

### PR DESCRIPTION
**Pull request by claudio4j**
https://issues.redhat.com/browse/ENTESB-17613

### CODE REVIEW

File: app/connector/kafka/pom.xml [hunk 1]
```diff
@@ -107,6 +107,26 @@
...
+        <exclusion>
+          <groupId>org.bouncycastle</groupId>
+          <artifactId>bcprov-jdk15on</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
+      <version>${bouncycastle.version}</version>
```
Comment (zregvart) [inline app/connector/kafka/pom.xml]:
```suggestion
      <version>${bouncycastle.version}</version>
      <scope>runtime</scope>
```

File: app/connector/kafka/pom.xml [hunk 2]
```diff
@@ -107,6 +107,26 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcprov-jdk15on</artifactId>
+      <version>${bouncycastle.version}</version>
```
Comment (zregvart) [inline app/connector/kafka/pom.xml]:
```suggestion
      <version>${bouncycastle.version}</version>
      <scope>runtime</scope>
```

**Review by zregvart — COMMENTED** [review]
Seems that the BouncyCastle dependencies in kafka connector need to be declared with runtime scope:

```
[INFO] --- maven-dependency-plugin:3.1.2:analyze-only (basepom.default) @ connector-kafka ---
Warning:  Unused declared dependencies found:
Warning:     org.bouncycastle:bcprov-jdk15on:jar:1.67:compile
Warning:     org.bouncycastle:bcpkix-jdk15on:jar:1.67:compile
```

File: app/connector/kafka/pom.xml (hunk 1 above)
Comment (claudio4j) [reply app/connector/kafka/pom.xml]: Looking into it now. Thanks.

**Pull request merged**